lib/web/apiserver.go

```diff
@@ -353,6 +353,7 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	h.DELETE("/webapi/mfa/token/:token/devices/:devicename", httplib.MakeHandler(h.deleteMFADeviceWithTokenHandle))
 	h.GET("/webapi/mfa/token/:token/devices", httplib.MakeHandler(h.getMFADevicesWithTokenHandle))
 	h.POST("/webapi/mfa/token/:token/authenticatechallenge", httplib.MakeHandler(h.createAuthenticateChallengeWithTokenHandle))
+	h.POST("/webapi/mfa/token/:token/registerchallenge/:deviceType", httplib.MakeHandler(h.createRegisterChallengeWithTokenHandle))
```

Comment:
I would place :deviceType in the body of the POST, so creating a registerchallenge has no params (like its sibling, the authenticatechallenge)